compiler/ballerina-lang/src/main/java/io/ballerina/projects/PackageManifest.java

```diff
@@ -163,8 +177,8 @@ public TopLevelNode getValue(String key) {
         return keywords;
     }
 
-    public List<String> exported() {
-        return exported;
+    public List<String> export() {
```

Comment:
I don't think it is correct to use `export` here. It should be `exportedModule`. It is okay to have different terms in `Ballerina.toml` and PackageDescriptor.

In `Ballerina.toml`, you say I want to export these modules, but in `PackageDescriptor`, you return a list of exported Modules.

Reply:
Made it `exportedModules`